src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Commands.RecoveryServices.Backup.Cmdlets.csproj

```diff
@@ -43,7 +43,7 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\packages\Microsoft.Azure.Common.2.1.0\lib\net45\Microsoft.Azure.Common.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Azure.Management.RecoveryServices.Backup, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+    <Reference Include="Microsoft.Azure.Management.RecoveryServices.Backup, Version=0.9.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
All other dependencies are using 1.0 ..  to be consistent is it ok if we change this to 1.0 ?